easybuild/easyconfigs/m/Molcas/Molcas-8.0-15.06.18_CentOS_6.6_x86_64.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'PackedBinary'
+
+name = 'Molcas'
+version = '8.0-15.06.18'
+versionsuffix = '_CentOS_6.6_x86_64'
+
+homepage = 'http://www.molcas.org'
+description = """Molcas is an ab initio quantum chemistry software package
+developed by scientists to be used by scientists. The basic philosophy is is to
+be able to treat general electronic structures for molecules consisting of
+atoms from most of the periodic table. As such, the primary focus of the
+package is on multiconfigurational methods with applications typically
+connected to the treatment of highly degenerate states."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.kvant.kemi.uu.se/molcas/']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tgz']
+unpack_options = "--transform='s,^%(namelower)s-%(version)s%(versionsuffix)s,%(namelower)s_files,'"
```

Comment:
@valtandor: https://github.com/valtandor/easybuild-easyconfigs/pull/4 fixes the remarks, except for this one

I'm not sure why this needs to be included, can you add a comment on the line above this one to explain why this is needed?